modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -59,6 +59,20 @@
      */
     COORDINATION_THRIFT_SERVER_PORT
             ("coordination/thriftServerPort", "7611", Integer.class),
+
+    /**
+     * The IP of the Thrift server to be advertised to external. This is used in NAT/containerized
+     * environments which has a local private IP and a routable public IP.
+     */
+    COORDINATION_THRIFT_SERVER_ADVERTISED_HOST
+            ("coordination/thriftServerAdvertisedHost", "", String.class),
```

Comment:
The reason is that thriftServerAdvertisedHost is optional, we do not have default values for it. Its not the IP that thrift server binds to, which is a mandatory property, but rather the advertised IP.